Add links to Matter and HomeKit integrations

c0ffeeca7 opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
- add links to Matter and HomeKit integrations to make it clearer that for the setup, either of these integrations is needed
- add subtitles to provide a bit of an overview



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

c0ffeeca7 committed:
Add links to Matter and HomeKit integrations

frenck commented on source/_integrations/thread.markdown:
```diff
@@ -16,12 +16,30 @@ ha_zeroconf: true
 The Thread integration helps you track the different Thread networks in your home and helps you manage their credentials. It is currently a work in progress.
 
-Thread is a low-power mesh networking standard which allows users to connect their devices within a home network. It uses the same RF technology as Zigbee, but provides IP connectivity similar to Wi-Fi. Unlike Zigbee, Thread does not allow to control devices directly: It is just a communication protocol. A higher level protocol such as Matter or HomeKit is required to control Thread enabled devic ...
+##### A networking standard and communication protocol - not a device control standard
```
Does the header order/level feel out of order? The page title is header 1; this is below it, but is header level 5?

frenck commented on source/_integrations/thread.markdown:
```diff
@@ -16,12 +16,30 @@ ha_zeroconf: true
 
-Thread is a low-power mesh networking standard which allows users to connect their devices within a home network. It uses the same RF technology as Zigbee, but provides IP connectivity similar to Wi-Fi. Unlike Zigbee, Thread does not allow to control devices directly: It is just a communication protocol. A higher level protocol such as Matter or HomeKit is required to control Thread enabled devic ...
+##### A networking standard and communication protocol - not a device control standard
+Thread is a low-power mesh networking standard which allows users to connect their devices within a home network. Thread allows devices to communicate with each other without the need for a central controller. This makes it ideal for home automation, where a large number of devices may need to be connected. Thread uses the same RF technology as Zigbee, but provides IP connectivity similar to Wi-F ...
```
```suggestion
Thread is a low-power mesh networking standard that allows users to connect their devices within a home network. Thread allows devices to communicate with each other without the need for a central controller. This makes it ideal for home automation, where a large number of devices may need to be connected. Thread uses the same RF technology as Zigbee but provides IP connectivity similar to Wi-Fi. Unlike Zigbee, Thread does not allow to control devices directly: It is just a communication protocol. A higher-level protocol, such as Matter or HomeKit, is required to control Thread-enabled devices. To see which home automation standard is supported by your device, check the icon on the packaging:
```

frenck commented on source/_integrations/thread.markdown:
```diff
@@ -16,12 +16,30 @@ ha_zeroconf: true
+##### A networking standard and communication protocol - not a device control standard
+Thread is a low-power mesh networking standard which allows users to connect their devices within a home network. Thread allows devices to communicate with each other without the need for a central controller. This makes it ideal for home automation, where a large number of devices may need to be connected. Thread uses the same RF technology as Zigbee, but provides IP connectivity similar to Wi-F ...
 
+<table>
+  <tr>
+    <th style="width:20%">Logo</th>
+    <th style="width:80%">Supported Home Automation standard</th>
+  </tr>
+  <tr>
+    <td> <img src="https://brands.home-assistant.io/_/matter/icon.png"  width="50"></td>
+    <td><a href="/integrations/matter/">Matter</a></td>
+  </tr>
+  <tr>
+    <td><img src="https://brands.home-assistant.io/_/homekit/icon.png"  width="50"></td>
+    <td><a href="/integrations/homekit/">HomeKit</a></td>
+  </tr>
+</table>
```
We should avoid the use of HTML in Markdown as much as possible.

Tables are available in Markdown.

c0ffeeca7 replied:
Unfortunately, I was not able to resize the columns using markdown. The method using the colon did not work for me in the preview. As a workaround, I shortened the title so that it fits into the 50% column.

frenck commented on source/_integrations/thread.markdown:
```diff
@@ -16,12 +16,30 @@ ha_zeroconf: true
+</table>
+
+
+##### Thread Border Routers connect the Thread network to Home Assistant
```
Same issue with the heading as pointed out above.

c0ffeeca7 committed:
Fix grammar

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

c0ffeeca7 committed:
Fix header hierarchy and md table

c0ffeeca7 committed:
Add subheading

- move table further up to make it clear from the get-go that devices are not added by the Thread integration but via the Matter or HomeKit integration

c0ffeeca7 committed:
Merge branch 'current' into thread-integration-add-links-and-titles

frenck commented on source/_integrations/thread.markdown:
```diff
@@ -16,22 +16,27 @@ ha_zeroconf: true
+### Vendor specific Thread networks
 
-Home Assistant communicates with TBRs over the local network. This means that TBRs do not have to be physically attached to Home Assistant to be used. You might already have a TBR as part of other products in your home, like a Google Nest Hub Gen 2 or Apple HomePod Mini. Each vendor forms their own network when you start using their products, so you can end up having a Home Assistant, an Apple, a ...
+Each vendor forms their own network when you start using their products, so you can end up having a Home Assistant, an Apple, and a Google Thread network in your home. These are all separate networks using different credentials, which prevents devices to roam between TBRs.
```
```suggestion
Each vendor forms their own network when you start using their products, so you can end up having a Home Assistant, an Apple, and a Google Thread network in your home. These are all separate networks using different credentials, which prevents devices from roaming between TBRs.
```

c0ffeeca7 committed:
Fix grammar

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

frenck reviewed (approved):
Thanks, @c0ffeeca7 👍 

../Frenck
